docs/source/user-manual/copy-paste-blocks.md

```diff
@@ -0,0 +1,43 @@
+---
+myst:
+  html_meta:
+    "description": "Copy, cut and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:description": "Copy, cut and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:title": "How to copy, cut, and paste Volto blocks"
+    "keywords": "Volto, Plone, frontend, React, User manual, edit blocks, copy, cut, paste"
+---
+
+(copy-cut-paste-blocks-label)=
+
+# Copy, Cut, and Paste blocks in Volto
+
+Volto supports copying/cutting of blocks from one page and pasting them to another page. Here's how you can do:
+
+(copy-cut-blocks-label)=
+
+## Copy/Cut blocks
+
+While in edit mode you can hold <kbd>ctrl</kbd> key to select multiple blocks you want to copy/cut.
+Once blocks are selected, you will have a copy <i class="fas fa-copy"></i> and a cut <i class="fas fa-cut"></i> option at the bottom of Toolbar.
+
+Additionally, Volto also has a hidden feature for selecting multiple blocks in sequence.
+This feature can be used by selecting a start block and an end block while holding the <kbd>shift</kbd> key.
+This will select all the blocks between the start and end block, allowing you to copy or delete multiple blocks at once.
+
+```{image} ../_static/user-manual/blocks/block-copy-cut.gif
```

Comment:
Animations are poor for usability, accessibility, and file size. We should instead use videos that have full playback capability and may be loaded async. I admit I let a few sneak in before, but I now realize we should resolve to improve the usability.

Also the recordings get reduced to a narrow viewport width of 686 pixels on desktop, and less on mobile. When recording we should try to keep the browser width, or the recorded area, to 680 pixels maximum.

mp4 is a well supported video format.

Here's the Sphinx extension that needs to be installed.

https://github.com/sphinx-contrib/video

Reply:
Thanks for the input! I thought the same, so I'll re-record it and upload the video with MP4 extension installed.

Reply:
@stevepiercy, Do you have any suggestions on how I can improve the animations in the video?

Reply:
Before I start recording, I usually resize the browser window to as narrow as possible to reduce unused space and focus attention on the action.

I took two screenshots. Note that I have clicked the vertical handle to collapse the right panel to make more space.

Note that the copy/cut toolbar is at the top of the browser window for 680 pixels, and drops to the side at 768 pixels. We need to be mindful of different device viewport sizes.

## 680 pixels wide

![Screen Shot 2023-02-03 at 5 33 57 AM](https://user-images.githubusercontent.com/102112/216616227-4884a49f-5d02-45af-bc36-922b51931352.png)

## 768 pixels wide

![Screen Shot 2023-02-03 at 5 38 04 AM](https://user-images.githubusercontent.com/102112/216617039-786adc28-cc16-40e9-9c5f-44fd15185b76.png)

I think that setting the browser width to 768 pixels will accommodate the largest audience of users. I doubt most users will edit with a phone or tablet, but we can always revisit that audience in future updates. I would focus on the desktop audience for now.

We also might update our documentation theme to accommodate wider layouts for the main content area, but that is another future consideration, not something to consider now.

Reply:
Thanks for the update! , I'll implement the suggested changes soon.
> We also might update our documentation theme to accommodate wider layouts for the main content area, but that is another future consideration, not something to consider now.

Ohh.. sure love to look into it in near future :)